webapp/src/components/sidebar_right/github_items.tsx

```diff
@@ -185,13 +188,28 @@ function GithubItems(props: GithubItemsProps) {
         if (item.status) {
             switch (item.status) {
             case 'success':
-                status = (<span style={{...style.icon, ...style.iconSucess}}><TickIcon/></span>);
+                status = (
+                    <span
+                        title={'Success'}
+                        style={{...style.icon, ...style.iconSucess}}
+                    ><TickIcon/></span>
```

Comment:
Can we format the JSX for this block and the two blocks below like this?

```suggestion
                    <span
                        title={'Success'}
                        style={{...style.icon, ...style.iconSucess}}
                    >
                        <TickIcon/>
                    </span>
```

Reply:
@maisnamrajusingh By "and the two blocks below" I meant to apply this change to the two blocks below this block as well. Can you please apply the change to the other two `span` blocks?

Reply:
sure, done.